### ISSUE TITLE
Print out pending test information gone from new version of Cedar.

### ISSUE THREAD

**Issue by pivotal-domenic-curro** [pivotal/cedar#303]
I've updated Cedar, and I no longer receive a 'P' during test run nor do I receive information about the pended test (one that I have places an 'x' in front of).

**Comment by tjarratt** [comment]
Hey @pivotal-domenic-curro and @xtreme-gavin-enns I've been on a couple few iOS projects in the SF office since you filed this issue and I haven't seen it occur with CocoaPods on the three codebases I worked on. Do you think we can close this out? Is there still something we can investigate and possibly fix?

**Comment by akitchen** [comment]
Isn't this due to XCTest not supporting pending tests? I assume the difference in observed behaviors here is one of test bundle vs. spec suite app.

**Comment by pivotal-ryan-mcleod** [comment]
Yeah, based on conversations I've had with people around the office the problem is just with people using XCTest. As someone who prefers running their tests in a test bundle it would be nice if we could somehow still get the pending test output in the console when running test bundles, but it seems like this isn't really possible to do without pending tests showing up as "passing" in Xcode, which might be kind of problematic.

**Comment by akitchen** [comment]
We have a story for this in the backlog now; I'm not sure if it's a regression or not but we'll see if at least the cedar reporter can support this





https://www.pivotaltracker.com/story/show/99707426
